dipy/core/gradients.py

```diff
@@ -138,6 +138,8 @@ def __init__(self, gradients, big_delta=None, small_delta=None,
                                  + "Please provide a string, an array of "
                                  + "strings, or an array of exact b-tensors. "
                                  + "String options: 'LTE', 'PTE', 'STE', 'CTE'")
+        else:
+            self.btens = btens
```

Comment:
Maybe be explicit here?
```suggestion
            self.btens = None
```

Reply:
Thanks @arokem, updated accordingly.